py-glaredb/Cargo.toml

```diff
@@ -29,3 +29,7 @@ uuid = "1.4.1"
 async-trait = "0.1.72"
 anyhow = "1.0.72"
 once_cell = "1.18.0"
+
+[profile.release]
+codegen-units = 1
+lto = "fat"
```

Comment:
oh boy

Reply:
I can remove this if you want, kinda forgot it was in this PR. I wanted to run the benches with the fastest & smallest possible binary.

Reply:
Did you see a difference?

Reply:
I didn't compare them, but we probably should...